installer/roles/local_docker/templates/docker-compose.yml.j2

```diff
@@ -124,7 +128,15 @@ services:
       http_proxy: {{ http_proxy | default('') }}
       https_proxy: {{ https_proxy | default('') }}
       no_proxy: {{ no_proxy | default('') }}
+<<<<<<< HEAD
```

Comment:
Looks like this conflict didnt get resolved properly. Please use whatever is in devel as your source of truth.

Reply:
done, sorry about that.